src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,43 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { MovieType } from '.';
+
+type Props = {
+  movieData: MovieType
+};
+
+export const MovieCard: React.FC<Props> = ({ movieData }) => (
+  <>
```

Comment:
You have the same problem as you wrote to me))) Unnecesary element `<>`

Reply:
:) Yes, byt i match it when rewiev another solutions